spec/features/admin/page_editing_feature_spec.rb

```diff
@@ -111,6 +111,29 @@ class FooPreviewSource < Alchemy::Admin::PreviewUrl; end
             expect(page).to_not have_selector('input[type="checkbox"]#page_sitemap')
           end
         end
+
+        context "enable_searchable" do
+          let(:searchable) { true }
+
+          before do
+            Alchemy.enable_searchable = searchable
+            visit alchemy.configure_admin_page_path(a_page)
```

Comment:
We probably want to avoid side effects by resetting the `enable_searchable` to its default value after the spec run.